TheInternet/DropdownPage.cs

```diff
@@ -0,0 +1,26 @@
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+
+namespace TheInternet;
+
+public class DropDownPage : BasePage
+{
+    private SelectElement dropdown => new SelectElement(driver.FindElement(By.Id("dropdown")));
+    private By currentlySelectedOptionBy = By.CssSelector("option [selected='selected']");
```

Comment:
I said I'd get back to pseudo selectors:

This _might_ by an opportunity to use one, if it does work the selector would simply be `option:selected`.

Again this has a resilience benefit because there's lots of flexibility (read: sloppiness) in how you can specify selected in the html markup; You can actually use anything in the quotes (`<option selected="my-bottom">`) or it can have no value at all (`<option selected>`). 

Typical wordy extra bit from me: I believe the XHTML specs specified it must be `selected="selected"` so that's why you see it a fair bit, but XHTML was a bit of a strange concept that came and went.